### ISSUE TITLE
The `incorrect-number-of-attributes` works incorrectly when passing object as refence

### ISSUE THREAD

**Issue by maxonfjvipon** [objectionary/lints#694]
Consider this program:

```
# program.eo
[args] > program
  greeting > @
    QQ.io.stdout
    human QQ.io.stdin QQ.io.stdout
# greeting.eo
[out human] > greeting
  out > @
    QQ.txt.sprintf
      "Hello, %s!\n"
      * human.name
# human.eo
[in out] > human
  seq * > name!
    out "What is your name? "
    in.next-line
```

Here in the `program` we pass `QQ.io.stdout` to `greeting` as first argument to be used later. However the error is:

```
[ERROR] program:4 The object "Q.org.eolang.io.stdout" expects 1 arguments, while 0 provided (incorrect-number-of-attribu
tes (WPA))
```

**Comment by h1alexbel** [comment]
@maxonfjvipon thanks for reporting, let me investigate

**Comment by 0crat** [comment]
@maxonfjvipon Hello! Thank you for reporting a new bug in our project. Your contribution is valuable in improving our software quality. As per our team's policy, you've been awarded **12 points** for this report. We appreciate your attention to detail and encourage you to continue [reporting](https://www.yegor256.com/2018/04/24/right-way-to-report-bugs.html) any issues you may find. If you notice anything else that [doesn't look](https://www.yegor256.com/2018/02/06/where-to-find-more-bugs.html) right, please [don't hesitate](https://www.yegor256.com/2014/04/13/bugs-are-welcome.html) to let us know. Your current balance stands at [-80](https://www.eolang.org/zerocracy/objectionary-vitals.html). Remember to check your [Zerocracy balance](https://www.zerocracy.com/) as well. Keep up the great work!

**Comment by VasilevNStas** [comment]
@yegor256 @volodya-lombrozo @maxonfjvipon, this bug is **no longer reproducible on `master`**: the `incorrect-number-of-attributes` lint was part of the whole-program-analysis layer that has been completely removed in commit `235cda2d` (PR #845, Apr 2026). There is no `incorrect-number-of-attributes.xsl` in the repository anymore, so the incorrect behaviour with object-as-reference cannot occur.

I'd suggest closing this as resolved-by-refactoring.

**Comment by volodya-lombrozo** [comment]
@maxonfjvipon we don't have this lint either in `lints` nor in `wpa`. Closing this one. Thank your for reporting!
